import DataPrinter as dp
import exceptionExample as exceptionExample
import FileManager as fileManager

def main():

    # dp.printme("str")
    #
    # # PRINT LIST NUMBERS
    # listOfNumbers = ['2', '5', '7', '37', '67', '46', '745', '732', '734', '756']
    # dp.printFullList(listOfNumbers)
    # dp.printListFromTo(listOfNumbers, 3, 8)

    # fileManager.writeIntoFile("asen.txt","content to write")
    fileManager.deleteFile("asen.txt")



main()

